docs/reference/dropbox.md

```diff
@@ -0,0 +1,118 @@
+# Dropbox Connector
+
+The [Elastic Dropbox connector](../connectors/sources/dropbox.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+## Availability and prerequisites
+
+This connector is available as a **connector client** from the **Python connectors framework**. To use this connector, satisfy all [connector client requirements](https://www.elastic.co/guide/en/enterprise-search/master/build-connector.html).
+
+This connector is in **beta** and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.
+
+## Usage
+
+To use this connector as a **connector client**, use the **Customized connector** workflow.
+
+For additional operations, see [Usage](https://www.elastic.co/guide/en/enterprise-search/master/connectors-usage.html).
+
+## Get a Refresh Token
+
+### Create Dropbox OAuth App
+
+In order to authenticate via the Dropbox package, the user needs to configure an app key, app secret and refresh token to fetch the data from Dropbox. 
+
+To generate App Key and App Secret, user needs to follow below steps:
```

Comment:
Lines 21 and 23 can be removed.

Reply:
modified line no. 23 as `To generate App Key and App Secret, follow the next steps:` similar to this comment - https://github.com/elastic/connectors-python/pull/1104#discussion_r1243433804